### ISSUE TITLE
Braceless if/else do not have SCOPE

### ISSUE THREAD

**Issue by KatGarmash** [acheshkov/program_slicing#148]
Because there is not statement of type scope, certain methods do not work correctly. Case in point:
```
        public void methodEx() {
            if (cond())
                do1();
            else
                do2();
        }
   ```
extractions `do1();` and `do2();` are filtered because they have scope statement BRANCH, so this case is filtered by `__full_control_construction`

**Comment by KuYaki** [comment]
That is correct behavior (filter lonely "do1();") because it is not possible to extract "do1();" without breaking branch structure.

This will not be compiled:
```java
public void methodEx() {
    if (cond())
    else
        do2();
}
```
